unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -3389,22 +3610,66 @@ private IEnumerator scaleObject(Vector3 targetScale, SimObjPhysics target)
                         ItemInHand = null;
                     }
                 }
+                return true;
+            }
+            
+            target.transform.position = originalPos;
+            target.transform.rotation = originalRotation;
+            if (includeErrorMessage) {
+                SimObjPhysics hitSop = ancestorSimObjPhysics(colliderHitIfSpawned.gameObject);
+                errorMessage = (
+                    $"Spawn area not clear ({(hitSop != null ? hitSop.ObjectID : colliderHitIfSpawned.name)})" 
+                    + " is in the way), can't place object at that point"
+                );
+            }
+            return false;
+        }
+
+        public void PlaceObjectAtPoint(
```

Comment:
I'm not sure this is a good idea. `PutObject` explicitly places objects onto receptacles, while `PlaceObjectAtPoint` is not limited to receptacle typed objects, but rather anywhere where there is a upward facing surface below the "point" that the object is being placed at.

TeleportObject can move things in the air, but again `PlaceObjectAtPoint` automatically places the object at the correct height so it rests on the surface below the "point"

Reply:
Just from the names, I would expect the opposite. That is, `PlaceObjectAtPoint` is the one that floats the object at a point, while `TeleportObject` is the one that corrects the height so that the object rests on the surface.

Would a new `bool autoFixHeight` parameter in `TeleportObject` work better to depreciate `PlaceObjectAtPoint`?

Reply:
Yes I think that would work. I'm not sure if `autoFixHeight` is descriptive enough though. Perhaps something like `snapToSurface` might be more appropriate? I could see that working.

Reply:
Yeah, I like that better.

Reply:
I like this `snapToSurface` idea, really we should try to get as much functionality into the `TeleportObject` action as possible. I think this is beyond the scope of this PR though (especially since we want this done soon) so I will not do it for now.